Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM registry.access.redhat.com/ubi9/go-toolset:1.21.13-2.1729776560 as builder
 WORKDIR /build
 COPY --chown=1001:0 . .
-RUN make gobuild
+RUN make gobuild golint gotest
```

Comment:
`gobuild` will call `gotest`, we should reimplement checks in https://github.com/app-sre/go-qontract-reconcile/blob/main/pr_check.sh by merge

Reply:
Got it.

Reply:
the test structure is a little compex in this repo, `make update-schema` is to update `schema.graphql` file, which requires integration test like setup. `make validate-schema` just build `Dockerfile.validate` which is almost identical to `Dockerfile`. We can remove that file, by replacing usages at https://github.com/app-sre/go-qontract-reconcile/blob/6965aea62d84c37473740ee9f608828b3c7a9bc6/Makefile#L41-L43 to use `Dockerfile` and target `builder`.